templates/terraform/sweeper_file.go.erb

```diff
@@ -0,0 +1,115 @@
+<%= lines(autogen_notice :go) -%>
+
+package google
+
+import (
+  "fmt"
+  "testing"
+  "strings"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+<%
+sweeper_name = product_ns + object.name
+wrap_path = object&.nested_query&.keys&.first || object.collection_url_key
+-%>
+
+func init() {
+	resource.AddTestSweepers("<%= sweeper_name -%>", &resource.Sweeper{
+		Name: "<%= sweeper_name -%>",
+		F:    testSweep<%= sweeper_name -%>,
+	})
+}
+
+func testSweep<%= sweeper_name -%>(region string) error {
+	resource_name := "<%= sweeper_name -%>"
+	log.Printf("[SWEEP_LOG] Sweeping %s",resource_name)
```

Comment:
I noticed in the CI build at least, messages were not getting dropped if they didn't have the special keys. I thought this would be an easy way to filter out the tons of other logging that will be mixed in with the sweeper specific logs. Although I'm not to set on it if we want to keep consistency

Reply:
I think I'd rather the standard tags, since sweeper runs all happen before regular tests.